core/src/test/java/org/springframework/security/authentication/ProviderManagerTests.java

```diff
@@ -102,6 +102,26 @@ public void testStartupFailsIfProvidersNotSetAsVarargs() {
 		new ProviderManager((AuthenticationProvider) null);
 	}
 
+	@Test
+	public void testUsingNullNotPermittedList() {
```

Comment:
I think this test has the right idea. It seems like the list ought to have a `null` element, though, to fully describe the use case.

Reply:
I assumed it already existed.
I've created additional test case.